scipy/optimize/_qap.py

```diff
@@ -50,54 +50,44 @@ def quadratic_assignment(A, B, method="faq", options=None):
     options : dict, optional
         A dictionary of solver options. All solvers support the following:
 
-        partial_match : 2d-array of integers, optional, (default = None)
-            Allows the user to fix part of the matching between the two
-            matrices. In the literature, a partial match is also
-            known as a "seed" [2]_.
+        maximize : bool (default: False)
+            Setting `maximize` to ``True`` solves the Graph Matching Problem
+            (GMP) rather than the Quadratic Assignment Problem (QAP).
 
-            Each row of `partial_match` specifies the indices of a pair of
-            corresponding nodes, that is, node ``partial_match[i, 0]`` of `A`
-            is matched to node ``partial_match[i, 1]`` of `B`. Accordingly,
-            ``partial_match`` is an array of size ``(m , 2)``, where ``m`` is
-            not greater than the number of nodes.
+        partial_match : 2-D array of integers, optional (default: None)
+            Fixes part of the matching. Also known as a "seed" [2]_.
 
-        maximize : bool (default = False)
-            Setting `maximize` to ``True`` solves the Graph Matching Problem
-            (GMP) rather than the Quadratic Assingnment Problem (QAP).
-
-        rng : {None, int, `~np.random.RandomState`, `~np.random.Generator`}
-            This parameter defines the object to use for drawing random
-            variates.
-            If `rng` is ``None`` the `~np.random.RandomState` singleton is
-            used.
-            If `rng` is an int, a new ``RandomState`` instance is used,
-            seeded with `rng`.
-            If `rng` is already a ``RandomState`` or ``Generator``
-            instance, then that object is used.
-            Default is None.
+            Each row of `partial_match` specifies a pair of matched nodes:
+            node ``partial_match[i, 0]`` of `A` is matched to node
+            ``partial_match[i, 1]`` of `B`. The array has shape ``(m, 2)``,
+            where ``m`` is not greater than the number of nodes, :math:`n`.
```

Comment:
having :math:`n` right next to `.` might be causing the error:
`/home/circleci/repo/doc/source/optimize.qap-faq.rst:92: WARNING: Inline interpreted text or phrase reference start-string without end-string.`

![image](https://user-images.githubusercontent.com/6570539/93405842-fa5ab500-f842-11ea-9628-1aac53c95d5d.png)

@tylerjereddy I'm sorry if you're not the right person to ask, but do you know if we can get the .rst files uploaded as artifacts in CircleCI? It would be a lot easier to debug from line numbers.

Reply:
Presumably might be done with something similar to what we have for html:

```yaml
 57       - store_artifacts:
 58           path: doc/build/html-scipyorg
 59           destination: html-scipyorg
```

https://circleci.com/docs/2.0/artifacts/ 

If you mean taking `.rst` files from around the code base paths and just uploading them, could probably use some shell command or globbing to move them to a single path for upload maybe? I probably don't know enough about the debug scenario to understand the advantage as well as you I guess.

Reply:
Yes, I think something like that. The reason is that the documentation build errors reference line numbers, but those line numbers correspond with .rst files that we can't see. If we could find the lines, we'd have more context to figure out what to change in the docstrings.